drodrpg/DRODLib/Character.cpp

```diff
@@ -3340,6 +3350,7 @@ void CCharacter::Process(
 
 			case CCharacterCommand::CC_SetPlayerAppearance:
 			{
+				if (bRoomBeingDisplayedOnly) break;
```

Comment:
I'm wondering whether this check is better placed only in the else block, because it seems like it could work fine for an "if" condition check without harmful side-effects.

Reply:
I think that will be fine, although in general I don't know if I want checks on player state to work if the player isn't even in the room, but in this particular case I think this will be OK.